docs/docs/setup/jwt.md

```diff
@@ -21,22 +25,23 @@ secret_key = "jwt-key.pem"
 expiration_time = "3min"
 ```
 
-Currently, the only supported signing algorithm is `ES256`.
 The secret key has to be a key matching the algorithm.
 For `ES256`, that's an EC key encoded as PKCS#8.
-To generate such a key, you can run these commands:
+To generate such a key, you can run these commands (you can replace `secp256r1` with other supported values like `secp384r1`):
```

Comment:
I would put the parenthesized part after the actual command

Reply:
:shrug:

Reply:
:shrug: